tests/unit/components/labelPrinting/LabelPrintingModal.spec.js

```diff
@@ -0,0 +1,87 @@
+import LabelPrintingModal from '@/components/labelPrinting/LabelPrintingModal'
+import { localVue, mount, store } from 'testHelper'
+
+describe('Modal.vue', () => {
+  let wrapper, modal
+
+  beforeEach(() => {
+    wrapper = mount(LabelPrintingModal, {
+      localVue,
+      store,
+      propsData: {
+        barcodesList: ['aBarcode-L1', 'aBarcode-L2'],
+        printer: { value: 1, text: 'stub', type: 'squix' },
+        copies: '3',
+        disabled: true,
+        isStatic: true,
+      },
+    })
+    modal = wrapper.vm
+  })
+
+  it('will have an modal component', () => {
+    expect(wrapper.find('.modal')).toBeTruthy()
+  })
+
+  it('will have an button component', () => {
+    expect(wrapper.find('.btn').element).toBeTruthy()
+    expect(wrapper.find('.btn').text()).toEqual('Print Labels')
+  })
+
+  describe('modal', () => {
+    it('displays the barcodes to be printed name', () => {
```

Comment:
displays the barcodes to be printed name =>
displays the barcodes to be printed